githubteacher-orlando.md

```diff
@@ -0,0 +1,16 @@
+# Orlando, FL
+
+### Restaurants
+- Rocco's Tacos
+- Pharmacy
+
+### Theme Parks
+- Disney World
+- Universal Orlando
+
+### Local Hangouts
+- mini golfing
+
+### Beaches
+- East Coast
+- West Coast
```

Comment:
I do not like the gulf, I prefer East Coast beaches. Have you tried the beaches by Cape Canaveral?